Product/Production/Adapters/General/CONNECTAdapterPDPOpenSSO/src/main/java/com/sun/identity/xacml/plugins/XSPAXACMLAuthzDecisionQueryHandler.java

```diff
@@ -286,19 +284,18 @@ public XSPAXACMLAuthzDecisionQueryHandler() {
         }
 
         com.sun.identity.saml2.protocol.Response samlpResponse = createSamlpResponse(statement, status.getStatusCode()
-                .getValue());
+            .getValue());
 
         System.out.println("XSPAXACMLAuthzDecisionQueryHandler.handleQuery(), returning " + ":samlResponse=\n"
-                + samlpResponse.toXMLString(true, true));
+            + samlpResponse.toXMLString(true, true));
```

Comment:
@TabassumJafri Great catch, and I agree!! But, this is a special case -- "System.out/exit" calls will be taken out of as part of cleanup tasks surrounding SonarQube and CheckStyle during an upcoming sprint -- so, no need to add another commit for it now.  In the meantime, keep looking out for little things we can do to make the code better :)